### ISSUE TITLE
includeDirectories not working right?

### ISSUE THREAD

**Issue by depl0y** [Microsoft/vscode-npm-scripts#24]
The root of the project isn't the place where my package.json is stored. That is stored in `src/<project_name>`. 

I tried including that directory in `npm.includeDirectories`, but it doesn't want to perform an npm install from that directory. If I do not disable `npm.useRootDirectory`, it just performs the `npm install` twice in the root of my project.

If I set `npm.useRootDirectory = false`, then it only performs the `npm install` from the root of project, once.

```
"npm.includeDirectories": [
    "src/HTML5-front-end"
],
"npm.useRootDirectory": false
```

Also, does `includeDirectories` support the use of variables like `${workspaceRoot}` ?

When I run with the configuration above,the following error is shown:

`enoent ENOENT: no such file or directory, open '<root_path>/package.json'`, the `<root_path>` is actually the path to the root of my project, but I would think it would append the directory included in the configuration.

**Comment by egamma** [comment]
@depl0y I think I see what is going on, when running scripts in the integrated terminal then the directory for included directories isn't honored. As a work around you can try to run npm-scripts in the output window by setting: 
```json
npm.runInTerminal: false
```

**Comment by depl0y** [comment]
@egamma Yes, that seems to help. Now the packages seem to be installed correctly.

**Comment by ppluciennik** [comment]
I've prepared pull request (https://github.com/Microsoft/vscode-npm-scripts/pull/26) that should fix the problem and extend module to give option to run multiple scripts that have the same name (but located in different package.json).

**Comment by egamma** [comment]
Closed through PR #26 from @ppluciennik, thanks.
